evalml/tests/utils_tests/test_cli_utils.py

```diff
@@ -75,6 +76,20 @@ def test_print_deps_info(caplog, current_dir):
         assert requirement in out
 
 
+def test_get_deps_file(current_dir, tmpdir):
+    evalml_path = os.path.abspath(os.path.join(current_dir, "..", "..", ".."))
+    path = os.path.join(str(tmpdir), "requirements.txt")
+    get_evalml_requirements_file(evalml_path, path)
+    assert os.path.exists(path)
```

Comment:
Should we assert that the path didn't exist before the get call?

Also, it'd be great to test that the return value of the function matches the contents of the file as well!